### PR TITLE
Update seminaire-saint-sulpice-ecole-theologie.csl

### PR DESCRIPTION
**Pull request by guschartier**
Minor changes about footnote and update style, website

### CODE REVIEW

File: seminaire-saint-sulpice-ecole-theologie.csl [hunk 1]
```diff
@@ -7,26 +7,30 @@
...
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="fr">
     <terms>
-      <term name="cited"/>
-      <term name="page" form="short">
+        <term name="in">
+            <single>dans</single>
+            <multiple>dans</multiple>
+        </term>
```
Comment (rmzelle) [inline seminaire-saint-sulpice-ecole-theologie.csl]: Can you just use `<term name="in">dans</term>`? There is no need to define singular and plural versions.

**Comment by rmzelle** [comment]
You current

```
        <term name="in">
            <single>dans</single>
        </term>
```

should be

```
        <term name="in">dans</term>
```

**Comment by guschartier** [comment]
thanks

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**